torch_geometric/loader/neighbor_loader.py

```diff
@@ -212,6 +233,16 @@ class NeighborLoader(torch.utils.data.DataLoader):
         transform (Callable, optional): A function/transform that takes in
             a sampled mini-batch and returns a transformed version.
             (default: :obj:`None`)
+        time_attr (str, optional): An optional argument
+            to specify the node attribute that denotes the timestamps for
+            nodes in graph.
+            If set, data[time_attr] has to be a dictionary of type
```

Comment:
This is slightly off. Each node type sets its own `time_attr`, e.g., `data[node_type].time = ...`.